xbmc/addons/AddonEvents.h

```diff
@@ -15,7 +15,11 @@ namespace ADDON
   struct AddonEvent
   {
     std::string id;
+    uint32_t instance{0};
     explicit AddonEvent(std::string id) : id(std::move(id)) {}
+    explicit AddonEvent(std::string id, uint32_t instance) : id(std::move(id)), instance(instance)
```

Comment:
To my knowledge it does not make sense to use `explicit` for a ctor with no or more than one argument.